WordPress/src/test/java/org/wordpress/android/ui/bloggingprompts/promptslist/BloggingPromptsListViewModelTest.kt

```diff
@@ -0,0 +1,100 @@
+package org.wordpress.android.ui.bloggingprompts.promptslist
+
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.delay
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mock
+import org.mockito.internal.verification.Times
+import org.mockito.kotlin.doSuspendableAnswer
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.whenever
+import org.wordpress.android.BaseUnitTest
+import org.wordpress.android.ui.bloggingprompts.promptslist.model.BloggingPromptsListItemModel
+import org.wordpress.android.ui.bloggingprompts.promptslist.usecase.FetchBloggingPromptsListUseCase
+import org.wordpress.android.ui.bloggingprompts.promptslist.usecase.FetchBloggingPromptsListUseCase.Result.Failure
+import org.wordpress.android.ui.bloggingprompts.promptslist.usecase.FetchBloggingPromptsListUseCase.Result.Success
+import org.wordpress.android.util.NetworkUtilsWrapper
+import java.util.Date
+
+@ExperimentalCoroutinesApi
+class BloggingPromptsListViewModelTest : BaseUnitTest() {
+    @Mock private lateinit var fetchBloggingPromptsListUseCase: FetchBloggingPromptsListUseCase
+    @Mock private lateinit var tracker: BloggingPromptsListAnalyticsTracker
+    @Mock private lateinit var networkUtilsWrapper: NetworkUtilsWrapper
+    lateinit var viewModel: BloggingPromptsListViewModel
+
+    @Before
+    fun setUp() {
+        viewModel = BloggingPromptsListViewModel(
+                fetchBloggingPromptsListUseCase,
+                tracker,
+                networkUtilsWrapper,
+                testDispatcher()
+        )
+    }
+
+    @Test
+    fun `when start, then track screen seen event`() = test {
+        viewModel.start()
+        verify(tracker, Times(1)).trackScreenShown()
```

Comment:
Nitpick: would it make sense to change it to `verify(tracker).trackScreenShown` instead considering that the default check is already `times(1)`?

Reply:
That makes sense, removed!